src/backend/api/Fusion.Resources.Domain/Queries/GetResourceOwner.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Fusion.Integration;
+using Fusion.Integration.Profile;
+using MediatR;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Domain.Queries
+{
+    public class GetResourceOwner : IRequest<FusionPersonProfile?>
+    {
+        public GetResourceOwner(Guid personAzureUniqueId)
+        {
+            AzureUniqueId = personAzureUniqueId;
+        }
+
+        public Guid AzureUniqueId { get; set; }
+
+        public class Handler : IRequestHandler<GetResourceOwner, FusionPersonProfile?>
+        {
+            private readonly HttpClient client;
+            private readonly IFusionProfileResolver profileResolver;
+
+            public Handler(IHttpClientFactory httpClientFactory, IFusionProfileResolver profileResolver)
+            {
+                this.client = httpClientFactory.CreateClient("lineorg");
```

Comment:
clientname - constant or not ?

Reply:
this is a bit hacky.. should create integration lib or include in people api